packages/rfw/test/material_widgets_test.dart

```diff
@@ -146,4 +146,65 @@ void main() {
       skip: !runGoldens,
     );
   });
+
+  testWidgets('OverflowBar widget', (WidgetTester tester) async {
+    final Runtime runtime = Runtime()
+      ..update(const LibraryName(<String>['core']), createCoreWidgets())
+      ..update(
+          const LibraryName(<String>['material']), createMaterialWidgets());
+    final DynamicContent data = DynamicContent();
+    final List<String> eventLog = <String>[];
+    await tester.pumpWidget(
+      MaterialApp(
+        theme: ThemeData(useMaterial3: false),
+        home: RemoteWidget(
+          runtime: runtime,
+          data: data,
+          widget: const FullyQualifiedWidgetName(
+              LibraryName(<String>['test']), 'root'),
+          onEvent: (String eventName, DynamicMap eventArguments) {
+            eventLog.add('$eventName $eventArguments');
+          },
+        ),
+      ),
+    );
+    expect(tester.takeException().toString(),
+        contains('Could not find remote widget named'));
+
+    runtime.update(const LibraryName(<String>['test']), parseLibraryFile('''
+      import core;
+      import material;
+      widget root = Scaffold(
+        body: Card(
+          margin: [20.0],
+          child: Padding(
+            padding: [8.0],
+            child: OverflowBar(
+              spacing: 8.0,
+              children: [
+                ElevatedButton(
+                  onPressed: event 'button' { },
+                  child: Text(text: 'Elevated'),
+                ),
+                OutlinedButton(
+                  onPressed: event 'button' { },
+                  child: Text(text: 'Outlined'),
+                ),
+                TextButton(
+                  onPressed: event 'button' { },
+                  child: Text(text: 'Text'),
+                ),
+              ],
+            ),
+          ),
+        ),
+      );
+    '''));
+    await tester.pump();
+    await expectLater(
+      find.byType(RemoteWidget),
+      matchesGoldenFile('goldens/material_test.overflow_bar.png'),
+      skip: !runGoldens,
+    );
+  });
```

Comment:
learning from my mistake with ButtonBar, could we add more variants of this test with different argument values and so on?

Reply:
I just added another test for `OverflowBar` properties. This includes all the `OverflowBar` and tests in the regular  and overflow state.

When writing this test I found out `OverflowBar.clipBehavior` is simply passed but not used in the `OverflowBar` implementation. I'll file a bug for this and fix it.

Reply:
I also did a clean up to reduce code duplication.